src/cuga/backend/cuga_graph/nodes/cuga_lite/prompt_utils.py

```
@@ -4,13 +4,48 @@
+    if isinstance(val, bool):
+        return val
+    if isinstance(val, str):
+        return val.strip().lower() in ("true", "1", "yes", "on")
+    return bool(val)
+
+
+def few_shots_enabled_from_settings() -> bool:
+    """Whether MCP few-shots are enabled (prompt block + prefix messages); default True."""
+    try:
+        v = getattr(settings.advanced_features, "cuga_lite_enable_few_shots", True)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)